src/js/storage.js

```diff
@@ -99,11 +99,13 @@ BadgerPen.prototype = {
    * @returns {String} the presumed action for this FQDN
    **/
   getAction: function (domain, ignoreDNT) {
+    let checking_dnt = badger.isCheckingDNTPolicyEnabled();
```

Comment:
Could avoid making another variable by setting `ignoreDNT` to true if `badger.isCheckingDNTPolicyEnabled()` is false.